data/data/aws/vpc/vpc-public.tf

```diff
@@ -1,4 +1,6 @@
 resource "aws_internet_gateway" "igw" {
+  count = var.vpc ? 0 : 1
```

Comment:
what if the value is ""

Reply:
> what if the value is ""

I was hoping for something like Python's truthiness.  But turns out [Terraform doesn't like that][1], so now there are explicit `== null`.

[1]: //prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_installer/2438/pull-ci-openshift-installer-master-e2e-aws/7844